src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvokerCache.cs

```diff
@@ -165,13 +79,13 @@ public InnerCache(int version)
 
         private struct Entry
         {
-            public Entry(List<FilterItem> items, ObjectMethodExecutor executor)
+            public Entry(Func<ActionContext, IFilterMetadata[]> filterFactory, ObjectMethodExecutor executor)
```

Comment:
This is kinda gross. Why isn't this just some static method that gets passed the filter items?